libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -603,10 +603,10 @@ void NavEKF3_core::readBaroData()
 {
     // check to see if baro measurement has changed so we know if a new measurement has arrived
     // limit update rate to avoid overflowing the FIFO buffer
-    if (frontend->_baro.get_last_update() - lastBaroReceived_ms > frontend->sensorIntervalMin_ms) {
+    if (AP::baro().get_last_update() - lastBaroReceived_ms > frontend->sensorIntervalMin_ms) {
```

Comment:
I'd prefer we keep the same style in EKF2 and EKF3, either both get a variable or both don't.